Subject: add immediate binding when enabling position independent
  executables
Author: Steve Beattie <steve.beattie@canonical.com>

When enabling position independent executables (-pie) to get better
Address Space Layout Protection, using immediate binding (linking with
"-z now") gives better protection as well.

---
 makedefs |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/makedefs
+++ b/makedefs
@@ -1167,7 +1167,7 @@
        case " $CCARGS " in
          *" $CCARGS_PIE "*) CCARGS_PIE=;;
        esac
-       SYSLIBS_PIE="-pie";;
+       SYSLIBS_PIE="-pie -z now";;
 ""|no) ;;
     *) error "Specify \"pie=yes\" or \"pie=no\"";;
 esac
